test/functional/platform/Document.spec.js

```diff
@@ -221,7 +223,10 @@ describe('Platform', () => {
           replace: [storedDocument],
         }, identity);
       } catch (e) {
-        const [error] = JSON.parse(e.metadata.get('errors'));
+        expect(e.code).to.be.equal(2);
+        expect(e.message).to.be.equal('Invalid state transition: DocumentTimestampWindowViolationError: Duplicate Document found');
```

Comment:
Are you sure that the message is `Duplicate Document found` for `DocumentTimestampWindowViolationError` error name?

Reply:
Hm, didn't notice that, let me check